addons/common/functions/fnc_markerDecay.sqf

```diff
@@ -0,0 +1,28 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3
+ * Function starts marker decay loop with given time (in minutes) to decay by half.
+ *
+ * Arguments:
+ * 0: Marker <STRING>
+ * 1: Time to decay by half (in minutes) <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * ["my_marker"] call afsk_common_fnc_markerDecay
+ *
+ * Public: No
+ */
+
+params ["_marker", ["_decayHalfTime", -1]];
+
+if (_decayHalfTime isEqualTo -1) then {
+    _decayHalfTime = 10;
```

Comment:
I wanted to use some setting value in the future, but now I see that it would work the same if I just set some global setting variable as default. Will be fixed.